packages/common/stateInfo.ts

```diff
@@ -39,8 +39,9 @@ export interface BaseInfo extends Locale {
   oid: string
   ip?: string
   userAgent?: string
-  signature?: string
   idPhoto?: string
+  signature?: string
+  signatureUsedCanvas?: boolean
```

Comment:
Can we make this type `signatureType?: SignatureType` where `type SignatureType = 'upload' | 'canvas'` instead of a boolean?